src/components/v2/chartDeploymentHistory/ChartDeploymentHistory.component.tsx

```diff
@@ -457,6 +471,11 @@ function ChartDeploymentHistory({
                             </button>
                         </Tippy>
                     )}
+                    {showDockerInfo && (
+                        <VisibleModal className="" close={closeDockerInfoTab}>
```

Comment:
We should move this VisibleModal inside dockerListModal

Reply:
Done